src/v/config/configuration.cc

```diff
@@ -40,7 +40,7 @@ configuration::configuration()
      .example = "2147483648",
      .visibility = visibility::tunable},
     128_MiB,
-    {.min = 1_MiB})
+    {.min = 1_KiB})
```

Comment:
Should `log_segment_size_min` be updated too if we wanted to go down this path?

Reply:
you mean adding a {.min = 1_KiB} to log_segment_size_min?

Reply:
sounds good, i'll add it